charts/k8spacket/values.yaml

```diff
@@ -83,6 +88,35 @@ k8sPacket:
       enabled: true
       ## Hide source port when 'true' (set to string value 'dynamic' instead of decimal real source port) for Prometheus metrics cardinality reasons
       hideSourcePort: true
+      ## Prometheus Operator ServiceMonitor configuration
+      serviceMonitor:
```

Comment:
path `.Values.k8sPacket.*` is for variables provided to the application. 
I think it will be better to move `serviceMonitor` to the root path like: 
```
.Values.serviceMonitor.*
```

Reply:
hey @k8spacket,
I have made all the adjustments. Thanks a lot!